homeassistant/components/geo_json_events/config_flow.py

```diff
@@ -19,14 +19,22 @@
 from homeassistant.helpers import config_validation as cv, selector
 from homeassistant.util.unit_conversion import DistanceConverter
 
-from .const import DEFAULT_RADIUS_IN_M, DOMAIN
+from .const import (
+    CONF_UPDATE_INTERVAL,
+    DEFAULT_RADIUS_IN_M,
+    DEFAULT_UPDATE_INTERVAL,
+    DOMAIN,
+)
 
 DATA_SCHEMA = vol.Schema(
     {
         vol.Required(CONF_URL): cv.string,
         vol.Required(CONF_LOCATION): selector.LocationSelector(
             selector.LocationSelectorConfig(radius=True, icon="")
         ),
+        vol.Required(CONF_UPDATE_INTERVAL, default=DEFAULT_UPDATE_INTERVAL): vol.All(
+            vol.Coerce(int), vol.Range(min=60)
+        ),
```

Comment:
Hi there @vingerha 👋 

We do not allow integrations to configure a update interval.

Instead, we provide a sane default out of the box. If a user wants a custom schedule or interval, one can use the `home_assistant.update_entity` action.

../Frenck

Reply:
Clear but this integration does not allow that as it always defaults to 300s, so its present status it is already not adhering. I will have a look if this can be made more in-line

Reply:
Correction, it works now... had not looked at it for almost 2y so either it changed or I missed things back then. If I would have known this to work then I would not have spent time on a PR :(